prompt_toolkit/widgets/base.py

```diff
@@ -402,7 +409,8 @@ def get_style() -> str:
         )
 
     def _get_text_fragments(self) -> StyleAndTextTuples:
-        text = ("{:^%s}" % (self.width - 2)).format(self.text)
+        width = self.width - (len(self.left_symbol) + len(self.right_symbol))
```

Comment:
I'll merge it and fix this line.
Usage of ``len()`` is not correct, because it assumes that every character has a width of 1, which is not the case for double-width characters.